lib/models/file.js

```diff
@@ -27,28 +26,25 @@ async function getFile(fileId) {
     throw createError(400, 'Identifiant invalide')
   }
 
-  try {
-    // CHECK ID IN MONGO
-    const fileMeta = await mongo.db.collection('files').findOne({_id})
-    if (!fileMeta) {
-      throw new Error('<<FILE>> Fichier non trouvé dans la collection files')
-    }
-
-    // CHECK ID IN S3
-    const fileS3Exist = await s3Service.checkS3FileExists(_id.toHexString())
-    if (!fileS3Exist) {
-      throw new Error('<<FILE>> Fichier non trouvé sur S3')
-    }
-
-    // GET BUFFER FILE
-    const data = await s3Service.getS3File(_id.toHexString())
-    // CREATE AND RETURN OBJECT FILE
-    const file = {...fileMeta, data}
-    return file
-  } catch {
```

Comment:
Je ne comprends pas pourquoi tu as enlevé le try and catch ?

Reply:
Y'a plus le fallback "return mongo.getFile(_id)" donc si y'a une erreur au niveau de la récupération dans le S3 elle va remonter. Je vois qu'il y'a un try catch pour ça dans revision.js et par "w" dans server.js, les deux endroits où le getFile est appelé